cli/pcluster/cli.py

```diff
@@ -92,7 +93,7 @@ def config_logger():
     log_file_handler = logging.FileHandler(logfile)
```

Comment:
Could you please add a log file rotation? 
See: https://github.com/aws/aws-parallelcluster/blob/develop/cli/awsbatch/common.py#L351